carbon-design-system/carbon
# fix(Codesnippet): use v11 size in multiline snippet copy button

Closes #11277 

#### Changelog
- updates `field` to `sm` in codesnippet button

#### Testing / Reviewing

- View Codesnippet multi story 
- Copy button should look and work as expected
- All tests pass
